test/unit/annotation_spec.js

```diff
@@ -1552,6 +1570,74 @@ describe("annotation", function () {
       }
       promise.then(done, done.fail);
     });
+
+    it("should render auto-sized text for printing", function (done) {
```

Comment:
Can you also add a test `should render regular text for printing`, i.e., where the auto-size logic is not called and the font size from the default appearance is used? I think all cases are covered then: regular text, auto-sized text and password fields.